websites/R/Replit/dist/metadata.json

```
@@ -19,7 +23,7 @@
 		"nl": "Replit, voorheen Repl.it, is een in San Francisco gevestigde start-up en een online IDE (geïntegreerde ontwikkelomgeving) die 40 programmeertalen ondersteunt, waaronder Python, Lua, JavaScript, Julia, C++, C++11, C, C#, HTML/CSS/JS, Python met turtle en LOLCODE. Replit stelt gebruikers in staat om code te schrijven en apps te bouwen met behulp van een browser, zonder software te hoeven installeren op hun apparaten."
 	},
 	"url": "replit.com",
-	"version": "1.0.12",
+	"version": "1.0.13",
```

It's a feature addition, so it's more a `minor` change than a `patch`
```suggestion
	"version": "1.1.0",
```